test/configuration/test_cases/test_99_tcp.xml

```diff
@@ -4,7 +4,7 @@
     <!--
         This test verifies that:
         - the clients can locate each others and server's publisher and subscribers.
-        - create dynamic types using fast-RTPS xml schema.
+        - create dynamic types using Fast-DDS xml schema.
```

Comment:
The official brand name is `Fast DDS`. Please, ensure that no `Fast-DDS` reference exists in the repository (with the exception to the hyperlinks).